tests/test_pqserver.py

```diff
@@ -89,6 +168,176 @@ def test_root2(client):
+            "obs_end": "2024-05-28T22:19:05.395000",
+            "obs_end_mjd": 60458.929923548276,
+            "emulated": False,
+        },
+        7024052800011: {
+            "exposure_name": "CC_S_20240528_000011",
+            "controller": "S",
+            "day_obs": 20240528,
+            "seq_num": 11,
+            "physical_filter": "i_06",
+            "exp_midpt": "2024-05-28T22:19:00.632500",
+            "exp_midpt_mjd": 60458.929868432606,
+            "obs_start": "2024-05-28T22:19:00.086000",
+            "obs_start_mjd": 60458.929862109704,
+            "obs_end": "2024-05-28T22:19:01.179000",
+            "obs_end_mjd": 60458.92987475551,
+            "emulated": False,
+        },
+    }
+    response = lsstcomcamsim.post(
+        "/consdb/insert/lsstcomcamsim/exposure",
+        json={"obs_dict": data},
+    )
+    _assert_http_status(response, 200)
+    result = response.json()
+    assert "Data inserted" in result["message"]
+    assert result["table"] == "exposure"
+    assert result["instrument"] == "lsstcomcamsim"
+    assert result["obs_id"] == list(data.keys())
+
+
+def test_insert_multiple_update(lsstcomcamsim):
+    data = {
+        7024052800012: {
+            "exposure_name": "CC_S_20240528_000012",
+            "controller": "S",
+            "day_obs": 20240528,
+            "seq_num": 12,
+            "physical_filter": "i_06",
+            "exp_midpt": "2024-05-28T22:19:04.847500",
+            "exp_midpt_mjd": 60458.92991721521,
+            "obs_start": "2024-05-28T22:19:04.300000",
+            "obs_start_mjd": 60458.92991088214,
+            "obs_end": "2024-05-28T22:19:05.395000",
+            "obs_end_mjd": 60458.929923548276,
+            "emulated": False,
+        },
+        7024052800011: {
+            "exposure_name": "CC_S_20240528_000011",
+            "controller": "S",
+            "day_obs": 20240528,
+            "seq_num": 11,
+            "physical_filter": "i_06",
+            "exp_midpt": "2024-05-28T22:19:00.632500",
+            "exp_midpt_mjd": 60458.929868432606,
+            "obs_start": "2024-05-28T22:19:00.086000",
+            "obs_start_mjd": 60458.929862109704,
+            "obs_end": "2024-05-28T22:19:01.179000",
+            "obs_end_mjd": 60458.92987475551,
+            "emulated": False,
+        },
+    }
+    response = lsstcomcamsim.post(
+        "/consdb/insert/lsstcomcamsim/exposure",
+        json={"obs_dict": data},
+    )
+    _assert_http_status(response, 200)
+
+    data[7024052800012]["exposure_name"] = "fred"
+    data[7024052800011]["exposure_name"] = "sally"
+    response = lsstcomcamsim.post(
+        "/consdb/insert/lsstcomcamsim/exposure",
+        json={"obs_dict": data},
+    )
+    _assert_http_status(response, 500)
+
+    response = lsstcomcamsim.post(
+        "/consdb/insert/lsstcomcamsim/exposure?u=1",
+        json={"obs_dict": data},
+    )
+    _assert_http_status(response, 200)
+    assert response.json()["obs_id"] == list(data.keys())
```

Comment:
Is the ordering guaranteed?  Maybe sort to make sure?
Might be nice to do a query to make sure that the data actually got updated.